com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -2123,6 +2154,44 @@ private void SyncTime()
 #endif
         }
 
+        internal struct DisconnectReasonMessage : INetworkMessage
```

Comment:
Can we keep message stuff located in the Messaging/Messages directory?

Reply:
will do